app/src/test/kotlin/com/wire/android/util/UriUtilTest.kt

```diff
@@ -38,6 +38,23 @@ class UriUtilTest {
         assert(expected == actual)
     }
 
+    @Test
+    fun givenLink_whenTheLinkStartsWithMailTo_thenReturnsTheSameLink() {
+        val input = "mailto:alice@wire.com"
+        val expected = "mailto:alice@wire.com"
+        val actual = normalizeLink(input)
+        assert(expected == actual)
+    }
+
+    @Test
+    fun givenLink_whenTheLinkIsWireDeepLink_thenReturnsTheSameLink() {
+        val input = "wire://access/?config=https://nginz-https.elna.wire.link/deeplink.json"
+        val expected = "wire://access/?config=https://nginz-https.elna.wire.link/deeplink.json"
+        val actual = normalizeLink(input)
+        assert(expected == actual)
```

Comment:
issue: this makes debugging harder

`assert` will only throw us an exception that doesn't explain what is wrong, something like "expected true and is false".

```suggestion
        assertEquals(expected, atual)
```

This way we get an exception showing what was expected, and what was the actual value. Like so:

```
Expected :://google.com
Actual   :https://://google.com
```